client/src/router/v2.ts

```diff
@@ -1,61 +1,65 @@
+import CameraSettingsView from 'views/v2/CameraSettingsView';
 import DashboardView from 'views/v2/DashboardView';
 import DownloadFilesView from 'views/v2/DownloadFilesView';
-import SensorStatusView from 'views/v2/SensorStatusView';
-import PowerModelView from 'views/v2/PowerModelView';
+import OptionsView from 'views/v2/OptionsView';
 import PowerMapView from 'views/v2/PowerMapView';
 import PowerModelCalibrationView from 'views/v2/PowerModelCalibrationView';
-import CameraSettingsView from 'views/v2/CameraSettingsView';
-import OptionsView from 'views/v2/OptionsView';
+import PowerModelView from 'views/v2/PowerModelView';
+import SensorStatusView from 'views/v2/SensorStatusView';
 import { RouteInfo } from './route';
 
-/** @type {RouteInfo[]} */
-export const routes = [
+/**
+ * component type is force casted to React.Component.
+ *
+ * V2 views will have to be converted to ts otherwise.
+ */
+export const routes: RouteInfo[] = [
   {
     name: 'Dashboard',
     path: '/v2/',
     exact: true,
-    component: DashboardView,
+    component: DashboardView as unknown as React.Component,
```

Comment:
Why do you need to type cast all of these?

Reply:
The `RouteInfo` type requires the `component` property to be a `React.Component`. If I don't typecast it, then I would have to convert the v2 views.